index.html

```diff
@@ -11861,9 +11861,14 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
+				<p>User agents MUST either prune <code>aria-hidden</code> nodes from the accessibility tree, or mark all <code>aria-hidden</code> nodes as hidden.</p>
+				<p>Authors MAY use <code>aria-hidden="false"</code> within an <code>aria-hidden="true"</code> subtree to unhide the visibly rendered content within the <code>aria-hidden="false"</code> subtree.</p>
+				<p class="note">This is a change from ARIA 1.2 and earlier versions in several ways: 1) <code>aria-hidden="false"</code> no longer un-hides content that is also hidden in CSS, as there are other techniques to present content only to AT users that works more consistently (e.g. positioning offscreen), and 2) <code>aria-hidden="false"</code> previously was a <code>noop</code> on (e.g., did not affect) visible content, and 3) <code>aria-hidden="false"</code> previously operated only on a single element. However, the <code>aria-hidden="false"</code> behavior of previous versions was never consistently implemented, understood or commonly used.</p>
+				<p>Authors MUST NOT place focusable elements within <code>aria-hidden</code> content. This is because the AT will have no information about the content, so it cannot be presented to the user with AT. User agents MUST repair this situation by ignoring the <code>aria-hidden</code> attribute on the ancestor of the focused element from that point forward. When this repair is made, user agents SHOULD inform authors through developer tools (if available).</p>
+				<p>Authors MUST NOT use <code>aria-hidden="true"</code> on the document element or body of the main document (e.g., <code>window.top</code>), where that would hide the entire accessibility tree. User agents MUST repair this situation by ignoring <code>aria-hidden="true"</code> when it would prevent all visible content from being available to ATs. Again, when this repair is made, user agents SHOULD inform authors through developer tools (if available). Notably, this situation does not occur when <code>aria-hidden="false"</code> is provided on a visible descendant, e.g. on a <code>role="dialog"</code>element, because that would generate some accessible content.</p>
 				<p class="note">Authors are advised to use extreme caution and consider a wide range of disabilities when hiding visibly rendered content from assistive technologies. For example, a sighted, dexterity-impaired individual might use voice-controlled assistive technologies to access a visual interface. If an author hides visible link text "Go to checkout" and exposes similar, yet non-identical link text "Check out now" to the accessibility API, the user might be unable to access the interface they perceive using voice control. Similar problems can also arise for screen reader users. For example, a sighted telephone support technician might attempt to have the blind screen reader user click the "Go to checkout" link, which they might be unable to find using a type-ahead item search ("Go to…").</p>
```

Comment:
Maybe we could improve this paragraph. I find it cumbersome to understand everything but the first 7 words.

Authors are advised to use extreme caution with aria-hidden. Misuse easily and often leads to inaccessible content. Starting with ARIA 1.3, several mitigations have been added to reduce the likelihood of this. However, aria-hidden is too powerful to make any mistakes with: many users have no way to perceive content that uses it, and therefore authors MUST avoid using aria-hidden on any content required for successful user journeys.

Reply:
I actually liked the examples here! Would prefer if they were kept.